runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -252,8 +253,8 @@ public class StreamingDataflowWorker {
   private final Counter<Integer, Integer> totalAllocatedThreads;
   private final Counter<Long, Long> outstandingBytes;
   private final Counter<Long, Long> maxOutstandingBytes;
-  private final Counter<Long, Long> outstandingBundles;
-  private final Counter<Long, Long> maxOutstandingBundles;
+  private final Counter<Integer, Integer> outstandingBundles;
```

Comment:
I'm not sure if dataflow supports changing counter type in this way.  Additionally the max may be set higher (effectively unbounded) to some value over Integer.MAX currently.  It seems safer for that reason to revert the change from Long->Integer and instead just bound to int range when populating the scaling proto (presuming that is what motivated this change).

Reply:
I reverted the Long->Integer change and just cast the value from the BoundedQueueExecutor as a Long since it is stored as an int under the hood.